client/src/lib/page.js

```diff
@@ -602,7 +611,7 @@ export class DropdownLink extends Component {
 
         const clsName = "dropdown-item" + (props.className ? " " + props.className : "")
         return (
-            <Link to={props.to} className={clsName}>{props.children}</Link>
+            <Link to={props.to} className={clsName} onClick={() => window.location.href=props.to}>{props.children}</Link>
```

Comment:
Why is this? I guess, this would make any DropdownLink refresh the client.

Reply:
Ready https://github.com/Mailtrain-org/mailtrain/pull/960/commits/b8088acfaec80d6b37df0233d40b006c6501f3c3